src/js/popup.js

```diff
@@ -161,6 +164,9 @@ function loadTemplateEditor (openTemplate = null) {
     // Load in the custom input IDs.
     chrome.runtime.sendMessage({JDTIfunction: 'getInputIDs'}, function (response) {
         if (response.data) {
+            (response.data).forEach(function (obj) {
```

Comment:
Instead of looping through each domain object or inputID object add adding a parameter to each, couldn't we just use the listID/type to determine which class to add.

Or better yet just pass in the class or part of the class name as another variable into the sandbox function and use it directly in the template. You'd avoid the loop here and the if statement in the template.